src/ostorlab/agent/kb/kb.py

```diff
@@ -3,7 +3,7 @@
 import json
 import pathlib
 
-from typing import Dict
+from typing import Dict, Optional, Union
```

Comment:
Please use the modern types instead. `dict`, | and | None.

Reply:
They are not supported in python3.9